pkgs/tools/admin/tigervnc/default.nix

```diff
@@ -91,6 +91,11 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  postPatch = ''
```

Comment:
The [`patchPhase`](https://github.com/jasom/nixpkgs/blob/65e228007de0dd09463c4f9d0231d40744555c8d/pkgs/tools/admin/tigervnc/default.nix#L25) is overridden, so the post hook won't get executed.